target/linux/ramips/mt7620/profiles/zbtlink.mk

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
Not sure if I got you right.

The zbtlink.mk is not required. The results I posted above are with the zbtlink.mk file deleted. The usb packages are only included if you select the "Default Profile". So, drop this file.